README.md

```diff
@@ -17,12 +17,58 @@ You can install the development version of MEAanalysis from [GitHub](https://git
 devtools::install_github("ankemt/MEAanalysis")
 ```
 
-## Example
+## Getting started
 
-This is a basic example which shows you how to solve a common problem:
+The MEAanalysis package is under active development. Currently, it can be used to create a tidy data object that aggregates data from baseline and exposure neural metrics files that are output files of the MEA device.
```

Comment:
```suggestion
This package is under active development. Currently, it can be used to create a tidy data object that aggregates data from baseline and exposure neural metrics output files.
```